.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you add a local ignore for VS Code's project directory? It'd be better if local editor settings didn't get merged into master.

Reply:
Sure thing. Sorry about that- I should have caught it.